pkg/services/ngalert/notifier/receivers.go

```diff
@@ -197,6 +199,39 @@ func (am *Alertmanager) TestReceivers(ctx context.Context, c apimodels.TestRecei
 	return newTestReceiversResult(testAlert, append(invalid, results...), now), nil
 }
 
+func (am *Alertmanager) GetReceivers(ctx context.Context) apimodels.Receivers {
+	var receivers []*notify.Receiver
+	receivers = append(receivers, am.receivers...)
+	var apiReceivers apimodels.Receivers
+	for _, rcv := range receivers {
+		// Build integrations slice
+		var apiIntegrations []*models.Integration
+		for _, integration := range rcv.Integrations() {
+			lastNotify, lastNotifyDuration, lastNotifyError := integration.GetReport()
+			apiIntegrations = append(apiIntegrations, &models.Integration{
+				LastNotifyDuration: lastNotifyDuration.String(),
+				LastNotify:         strfmt.DateTime(lastNotify),
```

Comment:
Is this timestamp the time of the last attempt, or the last successful notification?

Reply:
It's the time of the last attempt, regardless of the result

Reply:
I wonder if we should call it something else, or even have two separate fields: `LastAttempt` and `LastNotify`?

Reply:
I agree that some of these fields can be a bit misleading given what they are actually reporting. Even `LastError` doesn't actually report the last error that occurred, it reports the error on the last notify attempt. So flapping errors (like 429s) will cause flapping value here.

This is something we had discussed during design, I'm just wondering if there's a way to make this clearer either through field naming, comments, grafana docs, API docs, or something else.

Reply:
This is misleading indeed. A few ideas:
- `lastNotify` could be the timestamp for the last successful notification.
- `lastAttempt` could be the timestamp for the last attempt, regardless of the result (what we're currently doing for `lastError`).
- `lastError` is not the actual last error. Maybe this could change to `lastAttemptError` or something like that.